setups/fullnode/docker/README.md

```diff
@@ -24,13 +24,11 @@ wget https://github.com/iotaledger/iota/raw/develop/crates/iota-config/data/full
 
 ### Add Peers
 
-For `Testnet` or `Devnet` nodes, edit the `fullnode.yaml` file to include peer nodes for state synchronization. Append
-the following to the end of the current configuration:
+For `Testnet` or `Devnet` nodes, edit the `fullnode.yaml` file to include peer nodes for state synchronization. Add the following to the `p2p-config` section in the configuration:
 
 #### Devnet
 
 ```yaml
-p2p-config:
```

Comment:
imho not a good idea to remove the `p2p-config` because people won't understand how the yaml should be indented

Reply:
Yeah I thought about that too. I questioned what will be more likely. People not copying it correctly or people duplicating the p2p-config 😅.

Reply:
Maybe remove it and add a small note about indentation?

Reply:
Tbh that is even worst. Indenting yaml is complicated enough. When you copy without having the `p2p-config` container you have no clue how many white-spaces you need in front. Displaying the `p2p-config` allows you to compare to your configuration.

Reply:
Tbf if people are not smart enough to do a copy and paste I'm wondering if they should operate a node :trollface: 
But I will re-add it 👍